.github/workflows/regression-report.yml

```diff
@@ -296,8 +295,10 @@ jobs:
       - name: Cache Generated Config and Project Files
         uses: actions/cache@v4
         with:
-          path: ${{ env.GENERATED_CONFIGS_PATH }}
-          key: ${{ env.CACHE_KEY }}-generated-configs
+          path: |
```

Comment:
Whole step should be just 
```
- name: Save cache
        uses: actions/cache/save@v4
        with:
          path: .
          key: ${{ env.CACHE_KEY }}
```

As we just need to store current folder to cache.

Reply:
I tried that. but, i am struggling to retrieve the generated_config folder in 
```
      - name: Process local-based generated Config and Project File
        if: needs.handle_generated_config_bundle.result == 'success'
        env:
          DIFF_CONFIG_LINK: ${{ needs.handle_generated_config_bundle.outputs.config_link }}
          DIFF_PROJECTS_LINK: ${{ needs.handle_generated_config_bundle.outputs.projects_link }}
        run: |
          mkdir -p .ci-temp
          echo "Config link: $DIFF_CONFIG_LINK"
          echo "Projects link: $DIFF_PROJECTS_LINK"
          cp "$DIFF_CONFIG_LINK" .ci-temp/diff_config.xml
          cp "$DIFF_PROJECTS_LINK" .ci-temp/project.properties
```